tests/acceptance/test_long_covid_dsl.py

```diff
@@ -0,0 +1,375 @@
+from contextlib import contextmanager
+from datetime import date, datetime
+
+import pytest
+
+from databuilder import codelist
+from databuilder.backends import TPPBackend
+from databuilder.dsl import Cohort, categorise
+from databuilder.tables import (
+    clinical_events,
+    covid_test_results,
+    hospitalizations,
+    patient_addresses,
+    patient_demographics,
+    practice_registrations,
+)
+from tests.lib.tpp_schema import (
+    apcs,
+    ctv3_event,
+    negative_test,
+    organisation,
+    patient,
+    patient_address,
+    positive_test,
+    registration,
+    snomed_event,
+)
+from tests.lib.util import extract
+
+from .codelists import (
+    any_long_covid_code,
+    any_primary_care_code,
+    covid_codes,
+    ethnicity_codes,
+    post_viral_fatigue_codes,
+)
+
+
+@pytest.fixture(scope="module")
+def populated_database(database):
+    setup_data(database)
+    yield database
+
+
+@pytest.fixture
+def cohort(populated_database):
+    class AssertingCohort(Cohort):
+        def assert_results(self, **results):
+            dataset = extract(self, TPPBackend, populated_database)[0]
```

Comment:
I missed the `[0]` at first.  Suggest splitting over two lines:

```
dataset = extract(self, TPPBackend, populated_database)
row = dataset[0]
```

and then making assertions about `row`.

Reply:
Good. Thank you.